go.mod

```diff
@@ -20,7 +20,7 @@ require (
 	github.com/sirupsen/logrus v1.8.1 // indirect
 	github.com/stretchr/testify v1.7.0
 	github.com/tmc/grpc-websocket-proxy v0.0.0-20200427203606-3cfed13b9966 // indirect
-	github.com/vmware-tanzu/carvel-vendir v0.22.1-0.20210822004142-43142f6b60cd
+	go get github.com/vmware-tanzu/carvel-vendir
```

Comment:
is it right?

Reply:
i think @danielhelfand  meant to run command `go get github.com/vmware-tanzu/carvel-vendir` in your terminal in the repository.

Reply:
Ok got it now . Sorry misunderstood it.

Reply:
I think you will need to run the script under [`./hack/build.sh`](https://github.com/vmware-tanzu/carvel-kapp-controller/blob/22299af8dd26423d997c6fefd60d2ba1cd28591a/hack/build.sh) in order to vendor in the dependency as well. Then should be good.

Reply:
It may actually be easier to just run `go mod vendor` from your terminal though since the build script I just mentioned requires other tools to be installed besides go.